aws/templates/id/id_ecs.ftl

```diff
@@ -240,6 +240,18 @@
                     "Names" : "ContainerNetworkLinks",
                     "Type" : BOOLEAN_TYPE,
                     "Default" : false
+                },
+                {
+                    "Names" : "Placement",
+                    "Children" : [
+                        {
+                            "Names" : "Strategy",
+                            "Type" : STRING_TYPE,
+                            "Values" : ["daemon"],
```

Comment:
Need "" as one of the value if it is the default

Reply:
Fixed